TimeSeriesAnalysis.m

```diff
@@ -342,7 +368,8 @@
     northings = 1000.*northings;
   
     fprintf('...');
-      
+    %save('prelinreg_TRAN.m')
+    %load('prelinreg_TRAN.m')

```

Comment:
Unødvendige linjer.

Reply:
Disse linjer blev brugt til at debugge i forskellige stadier af koden. Så istedet for at indlæse alle variable op til denne linje kunne man bare loade dem ind, da jeg havde gemt dem på forhånd på dette stadie. Men linjerne er fjernet nu :-)